packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts

```diff
@@ -0,0 +1,42 @@
+// @group:suite
+// @retry=2
+
+describe('New accounts', () => {
+        beforeEach(() => {
+             // Launches Trezor emulator
+            cy.task('startEmu', { wipe: true });
+            // Initializes Trezor (random seed)
+            cy.task('setupEmu', { needs_backup: false });
+            cy.task('startBridge');
+            cy.viewport(1024, 768).resetDb();
+            cy.prefixedVisit('/');
+            cy.passThroughInitialRun();
+            cy.discoveryShouldFinish();
+        });
+    
+
+    it('Goes to accounts and verifies that the "New default Accounts natively.." is displayed:', () => {
+        cy.prefixedVisit('/accounts/');
```

Comment:
Doing this on line 12

```
cy.prefixedVisit('/');
```

and then 
```
cy.prefixedVisit('/accounts/');
```

has no added value. cy.prefixedVisit does full reload. You don't want to do that. I think you want to replace line 19 with 
```
cy.getTestElement('@suite/menu/wallet-index').click();
```

Reply:
cb0a3c4762e4b4db44b9de5a6c79d524fdc8e310 done